examples/carousel/carousel-1/carousel-1.html

```diff
@@ -0,0 +1,577 @@
+        <li><a href="https://github.com/w3c/aria-practices/projects/10">Related Issues</a></li>
+        <li><a href="../../../#carousel">Design Pattern</a></li>
+      </ul>
+    </nav>
+    <main>
+      <h1>Auto-Rotating Image Carousel Example</h1>
+      <p>
+        Comments on this example are tracked in
+        <a href="https://github.com/w3c/aria-practices/issues/458">issue 458.</a>
+      </p>
+      <p>
+        The following example implementation of the
+        <a href="../../../#carousel">carousel design pattern</a>
+        demonstrates features of the pattern that are essential to accessibility for carousels that automatically start rotating when the page loads.
+        For instance, Rotation stops when users move focus to any control or link in the carousel or hover the mouse over carousel content.
+        The accessibility features section that follows the example describes these features in detail.
+      </p>
+
+      <section>
+        <h2 id="ex_label">Example</h2>
+        <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
+        <div id="ex1">
+            <section id="myCarousel" class="carousel slide" aria-roledescription="carousel" aria-label="Highlighted television shows">
+              <button class="pause">Stop Carousel</button>
+
+              <div class="carousel-inner">
+
+                <a class="previous carousel-control"
+                   role="button"
+                   tabindex="0"
+                   aria-controls="myCarousel-items"
+                   aria-label="Previous Slide">
+                  <svg width="30" height="30" version="1.1" xmlns="http://www.w3.org/2000/svg">
+                    <polygon points="30 0 30 30 0 15"
+                        stroke="white" fill="white" stroke-width="2"/>
+                  </svg>
+                </a>
+
+                <a class="next carousel-control"
+                   role="button"
+                   tabindex="0"
+                   aria-controls="myCarousel-items"
+                   aria-label="Next Slide">
+                  <svg width="30" height="30" version="1.1" xmlns="http://www.w3.org/2000/svg">
+                    <polygon points="0 0 30 15 0 30"
+                        stroke="white" fill="white" stroke-width="2"/>
+                  </svg>
+                </a>
+
+                <div id="myCarousel-items" class="carousel-items" aria-live="off">
+
+                  <div class="carousel-item active" role="group" aria-roledescription="slide" aria-label="1 of 6" aria-describedby="carousel-label-1">
+
+                    <div class="carousel-image">
+                      <a href="#">
+                        <img src="images/amsterdamslide__800x600.jpg" alt="Walking Tour in Amsterdam" />
+                      </a>
+                    </div>
+
+                    <div class="carousel-caption">
+
+                      <h2>
```

Comment:
@jongund, please change the h2 elements inside the slides to h3 so they fit in the page IA.